app/demoservice/libs/demos.py

```diff
@@ -136,14 +142,32 @@ def start_demo(
             github_user,
             github_repo,
         )
-        if not _is_repo_collaborator(github_user, github_repo, github_sender):
-            logger.info(
-                "%s is not a collaborator of this repo", github_sender
-            )
-            return (
-                "User is not a collaborator of this repo. "
-                "Please start demo manually."
-            )
+        try:
+            if not _is_repo_collaborator(
+                github_user,
+                github_repo,
+                github_sender
+            ):
+                logger.info(
+                    "%s is not a collaborator of this repo", github_sender
+                )
+                return (
+                    "User is not a collaborator of this repo. "
+                    "Please start demo manually."
+                )
+        except GitHubError as ge:
+            # If user does not have permission to check repo collaborators
+            # an exception with the 403 code is expected.
+            if 403 == ge.code:
+                bot = _github_bot_username()
+                return (
```

Comment:
Why are you returning a tuple in this case? It looks like it used to simply return a string.

Also, if this app is running with python >= 3.6 (I've no idea if it is), you might want to consider using [f-strings](https://cito.github.io/blog/f-strings/).